android/app/src/e2eRelease/assets/airshipconfig.properties

```diff
@@ -0,0 +1,7 @@
+appKey = 55vypj0ARc6cN09MX7ogtQ
+appSecret = EsSaqbdLSvmyC6kSBFJCtQ
+inProduction = true
+
+# Notification Customization
+notificationIcon = ic_notification
+notificationAccentColor = #2EAAE2
```

Comment:
I'm going to further improve this tomorrow. I'll make the release properties file the default, and have the alternative sit under debug -- meaning that release is used by default and we don't need to duplicate it in both release and e2erelease

Reply:
This didn't end up working. As we're planning to inject the credentials soon, this isn't that important anyway.

Reply:
I am curious about our plan to inject credentials. I don't know how airship works and how "secret" the keys are, but I know that if you simply decompile an apk, you get access to the files in the assets folder and you can see these keys.

Reply:
Yeah, plus we don't even use ProGuard at the moment. We're talking about this plan, but it's in a privileged channel currently.